kitchen.openr.yml

```diff
@@ -0,0 +1,19 @@
+---
+driver:
+  name: vagrant
+  synced_folders:
+    - [".", "/opt/ipmininet"]
+  customize:
+    memory: 4096
+
+provisioner:
+  name: shell
+  command: /opt/ipmininet/util/build_vm.sh
```

Comment:
```suggestion
  command: /opt/ipmininet/util/build_vm.sh && sudo pytest -v /opt/ipmininet
```

Reply:
Kitchen supplies a config section (verifiers) for running tests https://kitchen.ci/docs/getting-started/writing-test/. Separating the provisioning and the tests. I suggest adding the tests there.